Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny4/gcd.dfy

```diff
@@ -177,7 +177,7 @@ method EuclidGcd(X: pos, Y: pos) returns (gcd: pos)
 // ------------------------------------------------------------------------------------------------------
 // The alternative definitions that follow allow the two cases in the GCD algorithm to look more similar.
 
-lemma GcdSubtractAlt(x: pos, y: pos)
+lemma {:resource_limit "150e6"} GcdSubtractAlt(x: pos, y: pos)
```

Comment:
Why is this now needed?

Reply:
That example was just incredibly brittle, and for things in that state it seems that _any_ change, even one that overall should be a simplification, can throw things off the rails. Changing things as simple as the filenames passed in as arguments to the `:captureState` attribute was enough to change resource use by an order of magnitude. I've refactored that test, and one other, to be less brittle, and they no longer require raised resource limits.

The remaining failures will, I believe, be fixed by brittleness reductions included in #4597, so let's merge that one first, update this PR, and see if everything is good.